pandas/sparse/sparse.pyx

```diff
@@ -71,7 +71,7 @@ cdef class IntIndex(SparseIndex):
     def check_integrity(self):
         """
         Only need be strictly ascending and nothing less than 0 or greater than
-        totall ength
+        total length
         """
         pass
```

Comment:
yeah looks like was for a possible impl. Certainly could refactor if you'd like.